java/steps/yaks-jms/pom.xml

```diff
@@ -0,0 +1,115 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+    Licensed to the Apache Software Foundation (ASF) under one or more
+    contributor license agreements. See the NOTICE file distributed with
+    this work for additional information regarding copyright ownership.
+    The ASF licenses this file to You under the Apache License, Version 2.0
+    (the "License"); you may not use this file except in compliance with
+    the License.  You may obtain a copy of the License at
+
+         http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+  -->
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <parent>
+    <groupId>org.citrusframework.yaks</groupId>
+    <artifactId>yaks-steps</artifactId>
+    <version>1.0.0-SNAPSHOT</version>
+    <relativePath>../pom.xml</relativePath>
+  </parent>
+  <modelVersion>4.0.0</modelVersion>
+
+  <artifactId>yaks-jms</artifactId>
+  <name>YAKS :: Steps :: JMS</name>
+
+  <properties>
+    <activemq.version>5.15.11</activemq.version>
+  </properties>
+
+  <build>
+    <plugins>
+      <plugin>
+        <groupId>org.apache.activemq.tooling</groupId>
+        <artifactId>activemq-maven-plugin</artifactId>
+        <version>${activemq.version}</version>
+        <configuration>
+          <fork>true</fork>
+          <systemProperties>
+            <property>
+              <name>log4j.configuration</name>
+              <value>log4j.properties</value>
+            </property>
+          </systemProperties>
+        </configuration>
+        <executions>
+          <execution>
+            <phase>process-test-resources</phase>
+            <id>broker</id>
+            <goals>
+              <goal>run</goal>
+            </goals>
+          </execution>
+        </executions>
+      </plugin>
+    </plugins>
+  </build>
+
+  <dependencies>
+    <dependency>
+      <groupId>io.cucumber</groupId>
+      <artifactId>cucumber-java</artifactId>
+    </dependency>
+
+    <dependency>
+      <groupId>com.consol.citrus</groupId>
+      <artifactId>citrus-core</artifactId>
+    </dependency>
+
+    <dependency>
+      <groupId>com.consol.citrus</groupId>
+      <artifactId>citrus-java-dsl</artifactId>
```

Comment:
This citrus-java-dsl dependency can be safely removed as it is not needed anymore in Citrus 3.0.0-M1

Reply:
I am facing compilation error without this dependency:

```
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-compiler-plugin:3.8.1:compile (default-compile) on project yaks-jms: Compilation failure: Compilation failure: 
[ERROR] /home/jbouska/go/src/github.com/citrusframework/yaks/java/steps/yaks-jms/src/main/java/org/citrusframework/yaks/jms/JmsSteps.java:[29,38] package com.consol.citrus.dsl.endpoint does not exist
[ERROR] /home/jbouska/go/src/github.com/citrusframework/yaks/java/steps/yaks-jms/src/main/java/org/citrusframework/yaks/jms/JmsSteps.java:[81,23] cannot find symbol
```